README.md

```diff
@@ -31,7 +31,14 @@ poetry run pytest -k test_testset_resolver --run-resolvers
 ```
 
 # Library usage
-You can use OpenRarity shell tool to generate json or csv outputs of OpenRarity scoring and ranks for any collections
+You can install open rarity as a [python package](https://pypi.org/project/open-rarity/)
+```
+pip install open-rarity
+```
+and directly use the models and rankers. Example of how to use the library lives in the scripts/ folder.
```

Comment:
What does directly  use the models and rankers mean? Should we put this in the gitbook developer documentation instead?

Reply:
like once you install the library you can do
from open_rarity import Collection